### PR TITLE
test/scrub: only instruct clean PGs to scrub

### PR DESCRIPTION
**Pull request by ronen-fr**
Recent changes to the scrub scheduling mechanism, especially regarding the 'must_scrub' flag, cause operator scrub commands issued on a not-clean PG to be rejected - and forgotten.

This commit changes the tests to issue a scrub command only after the target PG is clean.

### CODE REVIEW

File: qa/standalone/ceph-helpers.sh [hunk 1]
```diff
@@ -1572,6 +1572,20 @@ function test_is_clean() {
 
 #######################################################################
 
+##
+# Predicate checking if the named PG is in state "acrive+clean"
```
Comment (ljflores) [inline qa/standalone/ceph-helpers.sh:1576]:
```suggestion
# Predicate checking if the named PG is in state "active+clean"
```

**Review by ljflores — APPROVED** [review]
Fine with me except for a small nit.

**Comment by ljflores** [comment]
This should be run through the QA suite before merging to ensure the test is working.

File: qa/standalone/ceph-helpers.sh (hunk 1 above)
Comment (ronen-fr) [reply qa/standalone/ceph-helpers.sh:1576]: I'll fix that. Thanks

**Comment by ljflores** [comment]
jenkins test make check

**Comment by ronen-fr** [comment]
The 8aa84ad version is buggy. The test for a clean PG rejects active+clean+inconsistent.
I'll fix shortly, then retest thru Teuthology.

**Comment by ronen-fr** [comment]
Merging based on multiple Teuthology runs (I had to include the change in all other branches I tested).

**Pull request merged**